frontend/appflowy_flutter/lib/plugins/document/presentation/more/more_button.dart

```diff
@@ -1,84 +1,159 @@
+              final actions = [
+                const FontSizeAction(),
+                const Divider(height: 4),
+                DuplicateViewAction(view: widget.view, mutex: popoverMutex),
+                DeleteViewAction(view: widget.view, mutex: popoverMutex),
+                _MoreActionFooter(
+                  dateFormat: dateFormat,
+                  documentCounters: state.documentCounters,
+                  createdAt: state.createdAt,
+                ),
+              ];
+
+              return ListView.separated(
+                shrinkWrap: true,
+                padding: EdgeInsets.zero,
+                itemCount: actions.length,
+                separatorBuilder: (_, __) => const VSpace(4),
+                physics: StyledScrollPhysics(),
+                itemBuilder: (_, index) => actions[index],
               );
             },
-            child: FlowyButton(
-              text: FlowyText.regular(
-                LocaleKeys.moreAction_fontSize.tr(),
-                color: AFThemeExtension.of(context).textColor,
+            child: FlowyTooltip(
+              message: LocaleKeys.moreAction_moreOptions.tr(),
+              child: FlowyHover(
+                child: Padding(
+                  padding: const EdgeInsets.all(6),
+                  child: FlowySvg(
+                    FlowySvgs.details_s,
+                    size: const Size(18, 18),
+                    color: Theme.of(context).iconTheme.color,
+                  ),
+                ),
               ),
-              leftIcon: Icon(
-                Icons.format_size_sharp,
-                color: Theme.of(context).iconTheme.color,
-                size: 18,
-              ),
-              leftIconSize: const Size(18, 18),
-              hoverColor: AFThemeExtension.of(context).lightGreyHover,
             ),
-          ),
-        ];
+          );
+        },
+      ),
+    );
+  }
+}
+
+class _MoreActionFooter extends StatelessWidget {
+  const _MoreActionFooter({
+    required this.dateFormat,
+    this.documentCounters,
+    this.createdAt,
+  });
 
-        return ListView.separated(
-          shrinkWrap: true,
-          padding: EdgeInsets.zero,
-          itemCount: actions.length,
-          separatorBuilder: (_, __) => const VSpace(4),
-          physics: StyledScrollPhysics(),
-          itemBuilder: (_, index) => actions[index],
-        );
-      },
-      child: FlowyTooltip(
-        message: LocaleKeys.moreAction_moreOptions.tr(),
-        child: FlowyHover(
-          child: Padding(
-            padding: const EdgeInsets.all(6),
-            child: FlowySvg(
-              FlowySvgs.details_s,
-              size: const Size(18, 18),
-              color: Theme.of(context).iconTheme.color,
+  final UserDateFormatPB dateFormat;
+  final Counters? documentCounters;
+  final DateTime? createdAt;
+
+  @override
+  Widget build(BuildContext context) {
+    if (documentCounters == null && createdAt == null) {
```

Comment:
If this is null, it will return const SizedBox.shrink which is still a widget in the ListView. This means that there will be an extra VSpace(4) separator